libs/ledger-live-common/src/families/polkadot/config.ts

```diff
@@ -7,6 +8,16 @@ export const polkadotConfig: Record<string, ConfigInfo> = {
       status: {
         type: "active",
       },
+      sidecar: {
+        url: getEnv("API_POLKADOT_SIDECAR"),
+        credentials: getEnv("API_POLKADOT_SIDECAR_CREDENTIALS"),
```

Comment:
I've a just question about "API_POLKADOT_SIDECAR_CREDENTIALS". 
It seems that it is an empty string by default in env.ts.
I am wondering whether it is useful ? Where is this "credentials" used?

Reply:
It is used in staging. The staging information requires a token / basic auth header. In production, it is not used (yet).